SoftLayer/CLI/nas/detail.py

```diff
@@ -0,0 +1,18 @@
+import click
+import json
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer import utils
+
+NAS_MASK = 'id,username,serviceResourceBackendIpAddress'
```

Comment:
what does the mask do? limit the properties returned? If so I would name this `NAS_PROPERTIES`